### ISSUE TITLE
Statistical / learning methods & DSEO

### ISSUE THREAD

**Issue by joncison** [edamontology/edamontology#360]
These are used by DSEO (https://docs.google.com/spreadsheets/d/1ix1eKplAX9KHbBsA6uU7G95_7lD5Q9jGfEL_LV7dYAQ/edit#gid=1477753083) and we could model them in EDAM:
-  *Unsupervised learning* 
-  *Ensembl learning* 
-  *Neural networks* 
-  *Recommender system* 
-  *Probabilistic graphical model* (EDAM Data)
-  *Kernel methods*
-  *Reinforcement learning* 
-  *Supervised learning* 
-  *Active learning* 
-  *Bayesian methods* 
-  *Gaussian processes* 
-  *Markov processes* 
-  *Monte Carlo methods* 
-  *Knowledge representation* 
- *Graph analytics* (see also https://github.com/edamontology/edamontology/issues/359)

As new EDAM topic or operation concepts?  Or synonyms on existing (or perhaps some new) concepts?  If at all?

And what about the existing *Statistical calculation* (http://edamontology.org/operation_2238) - we already have similar sorts of concepts there, and (if that pattern holds) it implies we need *Learning method* or some such.

What troubles me quite a bit, is that the long-established pattern in EDAM is to focus on *what* is done, not *how*.  The above list - and also the *Statistical calculation* branch appears (at 1st glance) to break that a bit **but** if tools provide these as abstract methods, not tied to any particular type of data, then I guess it's fine.

Please advise what to do.

**Comment by veitveit** [comment]
This is indeed a tricky one. They are all relevant methods, particularly with the recent rise of machine learning. But, as you say, they are not telling _what_ e.g. an **Operation** does. 

The _Statistical calculation_ branch is, to my opinion, necessary as it covers very general data analysis concepts which could work as an own (such as _Standardization/normalization_). 
Maybe that is a way to draw the line for inclusion of such terms? If the term as a single one can describe an operation, then we take it, otherwise not.

On the **Topics** side, we have _Machine learning_ which covers about half of these terms and **Statistics and probability** which I consider a very relevant part of Computational Biology. We could move _Statistics_ up to as general branch and then think about whether to include any relevant children. But this could be difficult as there is statistics practically everywhere but not exclusively.

**Comment by joncison** [comment]
I agree with your rule of thumb - which to my mind, none of these concepts quite obey.

So for now I added all these as **narrowSynonym** of _Machine learning_ or _Statistics and probability_ with a couple going under _Mathematics_.  I think this will do, for now.  We can always convert these to sub-concepts, if we wanted in the future.

cc @matuskalas FHI

**Comment by bayjan** [comment]
Can this issue be open again considering also the issue https://github.com/edamontology/edam-bioimaging/issues/17? Thank you.

**Comment by matuskalas** [comment]
Thanks for the heads-up @bayjan!
Indeed, this needs to be fixed, namely in https://github.com/edamontology/edam-bioimaging/issues/17. About half of the listed terms is there as concepts, the rest has to be figured out.
After they're all fixed, they will need to be merged into the mainline EDAM, something we can do with the highest priority for the ML/AI concepts.
